api/client/container/restart.go

```diff
@@ -39,7 +40,7 @@ func NewRestartCommand(dockerCli *client.DockerCli) *cobra.Command {
 func runRestart(dockerCli *client.DockerCli, opts *restartOptions) error {
 	var errs []string
 	for _, name := range opts.containers {
-		if err := dockerCli.Client().ContainerRestart(context.Background(), name, opts.nSeconds); err != nil {
+		if err := dockerCli.Client().ContainerRestart(context.Background(), name, time.Duration(opts.nSeconds)*time.Second); err != nil {
```

Comment:
Do you think or do you know? Because you are wrong.  @pdalpra is correct, you need an explicit case because its not like a literal `1 * time.Second`, the value is already an int and hast to be cast.